Gemfile

```diff
@@ -2,5 +2,5 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'smart_proxy', :git => "https://github.com/theforeman/smart-proxy"
+  gem 'smart_proxy', :git => 'https://github.com/theforeman/smart-proxy', :branch => 'develop'
```

Comment:
Actually, not. Since smart-proxy now have only develop branch (master was deleted), `bundle` will fail when trying to run. So need to point it to develop branch (which also makes sense in development).